dash/development/component_generator.py

```diff
@@ -58,8 +58,9 @@ def generate_components(
     reserved_patterns = "|".join("^{}$".format(p) for p in reserved_words)
 
     os.environ["NODE_PATH"] = "node_modules"
+
     cmd = shlex.split(
-        "node {} {} {} {}".format(
+        "node {} \"{}\" \"{}\" {}".format(
```

Comment:
minor:  a more pythonic way is to use a single quote to avoid the escape 
e.g.` 'node {} "{}"'.format()`